README.md

```diff
@@ -6,11 +6,11 @@
   </a>
 </p>
 
-# NUI
+# Onyx
 
-A Vue.js UI component library and Design System created by [Schwarz IT](https://it.schwarz).
+A Design System and Vue.js UI component library created by [Schwarz IT](https://it.schwarz).
```

Comment:
We should also apply this change to:
- packages/components/README.md
- packages/components/package.json
- apps/docs/src/index.md
- The GItHub repository description

Reply:
Done, description will be updated